src/pykiso/lib/connectors/cc_process.py

```diff
@@ -134,7 +134,7 @@ def start(self, executable: Optional[str] = None, args: Optional[List[str]] = No
             encoding=self._encoding,
             cwd=self._cwd,
             env=self._env,
-        )
+        )  # nosec B602
```

Comment:
Why is there an error?

Reply:
Codefactor wants shell=False thats why i added the ignore.

Reply:
Please comment something like: `# Because we need to use shell for xxx reason, we accept the risk of a ....... , nosec B602`

Reply:
I would like to discuss this topic in the next sync